src/compile/mark/encode/aria.ts

```diff
@@ -45,7 +45,13 @@ export function description(model: UnitModel) {
   const channelDef = encoding.description;
 
   if (channelDef) {
-    return wrapCondition(model, channelDef, 'description', cDef => textRef(cDef, model.config));
+    return wrapCondition({
```

Comment:
FYI, no behavior change here. I just switch `wrapCondition` to use parameter argument and make it explicit that `aria` won't need the new `invalidValueRef` added.